generator/template/model_template.go

```diff
@@ -333,6 +339,14 @@ func toGoType(column metadata.Column) interface{} {
 		return pgtype.Int8range{}
 	case "numrange":
 		return pgtype.Numrange{}
+	case "bool[]":
+		return pq.BoolArray{}
+	case "integer[]", "int4[]":
+		return pq.Int32Array{}
+	case "bigint[]":
+		return pq.Int64Array{}
+	case "text[]", "jsonb[]", "json[]":
+		return pq.StringArray{}
```

Comment:
I'm fine using `pq`  types for one dimensional arrays. Maybe `pgtype` can be used for multi dimensional arrays.
Other types as well, time, date, etc...